apps/explorer/src/pages/validator/ApyCalulator.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ActiveValidator } from '~/pages/validator/ValidatorDataTypes';
+
+export function apyCalc(validators: ActiveValidator, epoch: number) {
```

Comment:
This shouldn't be within the `pages/` folder, that really should just exist for routing. This should be in `component/` somewhere. Also the filename casing implies this exports a component but it does not (probably lowercase it.

Also, do not use shorthand, call this `calculateAPY()` or something similar.